src/chart/generateCategoricalChart.js

```diff
@@ -1079,19 +1084,33 @@ const generateCategoricalChart = ({
 
     handleTouchMove = (e) => {
       if (e.changedTouches != null && e.changedTouches.length > 0) {
-        this.handleMouseMove(e.changedTouches[0]);
+        const { onTouchMove } = this.props;
```

Comment:
@alreadyExisted The function handleMouseMove isn't called here. Can the tooltip show when we touch the chart ?